packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -326,7 +324,7 @@ export async function startGeth(gethBinaryPath: string, instance: GethInstanceCo
   }
 
   if (validating) {
-    gethArgs.push('--mine', '--minerthreads=10', `--nodekeyhex=${privateKey}`)
+    gethArgs.push('--mine', '--minerthreads=10')
```

Comment:
Can you remove `--minerthreads=10` as well?

Reply:
Done

Reply:
might need to push a new commit?